files/zh-cn/web/api/navigator/getusermedia/index.md

```diff
@@ -96,7 +96,7 @@ if (navigator.getUserMedia) {
 
 > **警告：** 新代码应当使用 {{domxref("Navigator.mediaDevices.getUserMedia()")}} 替代。
 
-## 更多参见
+## 参见
 
```

Comment:
L104 should be fixed(taken care of upstream in https://github.com/mdn/content/pull/30491/)

Reply:
fixed